db/migrate/20170118232430_create_join_table_authentication_configuration_script_base.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJoinTableAuthenticationConfigurationScriptBase < ActiveRecord::Migration[5.0]
+  def change
+    create_table :authentication_configuration_script_bases do |t|
```

Comment:
This should be `authentications_configuration_script_bases` to reflect join table conventions. To fix this, just change that here and then do `self.table_name = authentications_configuration_script_bases` in the `AuthenticationConfigurationScriptBase` model, and all should still work fine.